governance-and-automation/automated-workflows/minimum-number-of-owners-admin.md

```diff
@@ -23,5 +25,12 @@ The **Edit Policy** dialog opens where you can:
 
 ![Edit Policy Dialog](../../.gitbook/assets/set-up-automated-workflows-min-owners-dialog.png)
 
+{% hint style="info" %}
+
+**Please note:** By default, the Minimum Number of Owners Orphaned policy vulnerability is detected when there are less than 2 active owners assigned to a workspace. If there are no active owners assigned to a workspace, the [Orphaned Workspaces](../automated-workflows/orphaned-resources-admin.md) vulnerability is detected instead. 
+
+If less than 2 owners are assigned to a workspace in order to resolve the Orphaned Workspaces vulnerability, Syskit Point will then detect the Minimum Number of Owners vulnerability.
```

Comment:
malo mi je nejasno na prvu
Mislim da bi bilo razumljivije da piše u stilu: If 1 owner is assigned to a workspace when resolving the Orphaned Workspaces vulnerability, Syskit Point will then detect the Minimum Number of Owners vulnerability.